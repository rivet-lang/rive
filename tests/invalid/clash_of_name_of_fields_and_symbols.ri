struct Xyz {
    a: int;

    func a() {}
}