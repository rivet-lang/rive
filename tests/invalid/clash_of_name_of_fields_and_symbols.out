tests/invalid/clash_of_name_of_fields_and_symbols.ri:2:5: error: struct `Xyz` has field and function named `a`
    2 |     a: int;
      |     ^~~~~~
rivet: error: could not compile module `clash_of_name_of_fields_and_symbols`, aborting due to previous error